### ISSUE TITLE
not able to capture image or video even from gallery

### ISSUE THREAD

**Issue by babitakapoor112** [ivpusic/react-native-image-crop-picker#1233]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.28.0
- react-native v0.61.5

### Platform
Tell us to which platform this issue is related

- iOS

### Expected behaviour

photos/videos should be recorded or should be fetched form gallery 

### Actual behaviour
I am getting following error while recording video or capturing image 
Error: Error creating folder 'undefined/Heirloom', error: Error Domain=NSCocoaErrorDomain Code=513 "You don’t have permission to save the file “Heirloom” in the folder “undefined”." UserInfo={NSFilePath=undefined/Heirloom, NSUnderlyingError=0x283d453e0 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}
    at Object.fn [as mkdir] (NativeModules.js:99)
    at Object.mkdir (fs.js:126)
    at eval (VM5 AddMemory.bundle:2323)
    at tryCallOne (core.js:37)
    at core.js:123
    at JSTimers.js:289
    at _callTimer (JSTimers.js:146)
    at _callImmediatesPass (JSTimers.js:194)
    at Object.callImmediates (JSTimers.js:458)
    at MessageQueue.__callImmediates (MessageQueue.js:407)

### Steps to reproduce

1. 
npm install --save react-native-image-crop-picker
cd ios && pod update
2. import ImagePicker from react-native-image-crop-picker
use code on button press
onPress={() => {
                var size = 0;
                 ImagePicker.openPicker({
                      mediaType: "video",
                      multiple: true,
                      maxFiles:10,
                 }).then((video) => {
                        console.log(video);

                 }).catch(e => {
                           //console.log(e);
                           //throw e;
                            console.log(JSON.stringify(e));
                         });
                }

3.

### Attachments
![image](https://user-images.githubusercontent.com/45659052/74536059-4bc55d80-4f5d-11ea-80ad-6f57562a0853.png)

// stacktrace or any other useful debug info

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by achimbaggins** [comment]
yes I faced the issue too, I'm facing it on iOS Platform and Android it works as expected
